MAUI/Popup/overview.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: About .NET MAUI Popup control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+--- 
+
+# MAUI Popup (SfPopup) Overview
+
+Popup layout provides a layout that allows users to display an alert message with customizable buttons or load any desired view inside a popup. It provides options to fully customize the popup with custom header, body, and footer capabilities, allowing you to display important information to use precisely how user want to.
+
+## Key features
+
+* Load complex views and layouts as content.
+* Display popups with minimalistic code by calling only a single method.
+* Choose from default layouts, such as one-button layout and two-button layout, to display simple and minimalistic information.
```

Comment:
* Choose from default layouts, such as one-button and two-button layouts, to display simple and minimalistic information.

Reply:
modified